java/src/Ice/src/main/java/com/zeroc/IceInternal/DispatchWorkItem.java

```diff
@@ -6,8 +6,8 @@
 
 //
 // A helper class for thread pool work items that only need to call user
-// callbacks. If a dispatcher is installed with the communicator, the
-// thread pool work item is executed with the dispatcher, otherwise it's
+// callbacks. If an executor is installed with the communicator, the
+// thread pool work item is executed with the executor, otherwise it's
 // executed by a thread pool thread (after promoting a follower thread).
 //
 public abstract class DispatchWorkItem implements ThreadPoolWorkItem, Runnable {
```

Comment:
What about renaming this class, ExecutorWorkItem or UpCallWorkItem.

Reply:
Renamed to RunnableThreadPoolWorkItem.